src/net.h

```diff
@@ -409,6 +409,9 @@ class CNode
     int64_t nTimeOffset;
     CAddress addr;
 
+    //! set to true if this node is ok with no message checksum
+    bool skipChecksum = false;
+
```

Comment:
for consistency sake, maybe set this value in the constructor  with all the others.

Reply:
done